src/hammer-vlsi/defaults.yml

```diff
@@ -797,19 +917,19 @@ sim.inputs:
   execute_sim: true
 
 sim.submit:
-  command: "${vlsi.submit.command}"
-  command_meta: lazysubst
-  settings: "vlsi.submit.settings"
-  settings_meta: lazycrossref
+  command:                "${vlsi.submit.command}"
```

Comment:
Is there a way to make these less redundant across the different tools? (No need to solve this now, but we should file an issue)